stable/appmesh-controller/values.yaml

```diff
@@ -122,6 +122,11 @@ stats:
 # Enable cert-manager
 enableCertManager: false
 
+# podDisruptionBudget for Appmesh controller
+pdb:
+  # minAvailable pods for appmesh-controller
+  minAvailable: 0
```

Comment:
Why '0'?? Should be at least '1'

Reply:
By default there was never any PodDisruptionbudget so had assigned to 0. But, yes I agree, it should be 1 instead so that we have at least 1 pod running.